pkg/backporter/backporter_test.go

```diff
@@ -161,6 +129,10 @@ func TestGetClonesHandler(t *testing.T) {
 	if err != nil {
 		t.Errorf("Error while cloning bug: %v", err)
 	}
+	clone, err := fake.GetBug(cloneID)
+	if err != nil {
+		t.Errorf("Error retreiving clone: %v", err)
```

Comment:
I don't think the remainder of the test is useful if we fail here, so using `t.Fatalf` to abort it here rather than `Errorf` seems appropriate

Reply:
Alright I have switched the Errorf s with Fatalf s where appropriate.